es

```diff
@@ -0,0 +1,284 @@
+                "disclaimer_clickwrap_input_label": "Aceptar políticas",
+                "disclaimer_clickwrap_error_required": "Debe leer y aceptar las políticas para continuar",
+                "disclaimer_dialog_heading": "Aceptar para continuar",
+                "disclaimer_dialog_description": "Para seguir registrándose, necesitamos que lea y acepte nuestras políticas",
+                "disclaimer": "Al continuar, acepta nuestras políticas",
+                "disclaimer_terms_of_use_link": "Términos de uso",
+                "disclaimer_privacy_link": "Política de privacidad"
+            },
+            "sms_verify_code_page": {
+                "page_title": "Autenticación multifactor | ${business_name}",
+                "heading": "Verificar número de teléfono",
+                "description": "Ingrese el código que acabamos de enviar a ${phone_number]",
+                "code_input_label": "Código",
+                "code_input_error_expired_code": "Su código de confirmación ha caducado, por favor solicite uno nuevo",
+                "code_input_error_incorrect_code": "Ingrese un código de confirmación válido",
+                "code_input_error_incorrect_code_length": "El código de confirmación debe tener 6 dígitos",
+                "code_input_error_resend_requested_too_soon": "Su código de confirmación ha caducado, por favor solicite uno nuevo",
+                "confirmation_code_input_error_unknown_error": "Error de autenticación",
+                "continue_button": "Verificar",
+                "didnt_receive_a_code_description": "¿No recibió un código?",
+                "didnt_receive_a_code_link": "Reenviar código",
+                "choose_another_method_link": "Elegir otro método"
+            },
+            "subscribe_form_widget": {
+                "page_title": "Suscribirse",
+                "heading": "Manténgase al tanto",
+                "description": "Escuche de nosotros sobre actualizaciones de productos, noticias y eventos",
+                "name_input_label": "Nombre",
+                "name_input_error_required": "Ingrese su nombre,",
+                "name_input_error_exceeds_max_length": "El nombre es muy largo,",
+                "email_input_label": "Correo electrónico",
+                "email_input_error_invalid": "Ingrese un correo electrónico válido,",
+                "email_input_error_required": "El correo electrónico es obligatorio,",
+                "email_input_error_duplicate_email_found": "Este correo ya ha sido registrado,",
+                "continue_button": "Registrarse",
+                "success_heading": "¡Realizado correctamente! Bienvenido.",
+                "credit_powered_by": "Desarrollado por"
+            },
+            "verification_code_sms": {
+                "description": "\"${generated_code} es tu código único para iniciar sesión en ${business_name}\n\n@ ${business_name} # ${generated_code]"
```

Comment:
@onderay we need to close the white space here `@ ${business_name} # ${generated_code]` to be `@${business_name} #${generated_code]`

Reply:
@DaveOrDead updated